src/pages/new/docs/configuration/client-auth/index.vue

```diff
@@ -4,6 +4,7 @@
       <div v-html="markdown1" />
       <div style="margin-bottom: 1rem" v-html="markdown2" />
       <div v-html="markdown3" />
+      <div v-html="markdown4" />
       <NEXUSe2eImage src="@/assets/force_client_cert.png" style="padding-top: 0"/>
       <div v-html="markdown4" />
```

Comment:
Wenn die variablen geändert werden, müssten die nicht auch in den divs erhöht werden von 4-6